crates/sparrow-session/src/expr.rs

```diff
@@ -29,4 +29,6 @@ pub enum Literal {
     Int64(i64),
     UInt64(u64),
     Float64(f64),
+    /// time in s, nanos
+    Timedelta(i64, i64),
```

Comment:
Make it a record and name the fields? `TimeDelta { seconds: i64, nanos: i64 }` is a lot clearer.